Backpack-SwiftUI/Price/Classes/BPKPrice.swift

```diff
@@ -100,36 +52,59 @@ public struct BPKPrice: View {
     }
     
     public var body: some View {
-        VStack(alignment: mapAlignment(), spacing: BPKSpacing.none) {
-            HStack(spacing: .sm) {
-                ForEach(topLabels(), id: \.self) { item in
-                    accessoryText(item)
-                        .strikethrough(item == previousPrice)
-                }
+        switch alignment {
+        case .leading:
+            VStack(alignment: .leading, spacing: BPKSpacing.none) {
+                buildContent()
+            }
+        case .trailing:
+            VStack(alignment: .trailing, spacing: BPKSpacing.none) {
+                buildContent()
             }
+        case .row:
+            HStack(alignment: .firstTextBaseline, spacing: .sm) {
+                buildContent()
+            }
+        }
+    }
+    
+    private func buildContent() -> some View {
```

Comment:
you can remove this Group thing if you tag this method with a @ViewBuilder annotation! also, since you're not passing any parameters, I'd use a `var` here instead. something like
```swift
@ViewBuilder
private var content: some View {
  additionalInfo
  switch .... {
     ....
  }
}